ubuntudesign/documentation_builder/operations.py

```diff
@@ -171,7 +171,7 @@ def parse_markdown(parser, template, filepath, metadata):
 
         metadata.update(markdown_meta)
 
-    if metadata.get('table_of_contents'):
+    if not metadata.get('table_of_contents') is False:
```

Comment:
so `if metadata.get('table_of_contents'):` ?

Reply:
I needed this as I wanted this to run even if it was Null. The only condition I didn't want this to run is when it was specifically set to False. I'm removing this functionally after talking with Robin.

Reply:
@anthonydillon `if metadata.get('table_of_contents') is not False` makes more sense.

But please see my question about why we're keeping this key at all?

Reply:
Good to know. Removed this logic.